### ISSUE TITLE
Default value for shuffle in train_test_split() is True - consider to use False

### ISSUE THREAD

**Issue by arcsi1989** [JamesTwallin/gb_grid_frequencies#3]
### Description
The current modelling approach creates training and test sets by:
`X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)`
However, the shuffle argument by default is `True`, resulting in mixed data in both the training and test sets. To ensure that the training data set does not see values from the test set and vice-versa, one could use the `shuffle=False` configuration.

### Steps to Reproduce

1. Set the shuffle=False in the train_test_split() 

### Expected Behaviour
The R^2 value changes depending on the assigned value (True or False) to the shuffle parameter. Especially for applications, where the signal level is low, the current variant with shuffle=True might create overly optimistic/high R^2 values on the mixed test-set.

### Labels
- [ ] bug
- [x] enhancement
- [ ] question
- [ ] documentation
- [ ] other

**Comment by JamesTwallin** [comment]
Setting shuffle=False doesn't create any additional separation between sets; it simply takes the first portion of the ordered data for training and the remaining portion for testing, 

I don't think it does as you're describing.
